CONTRIBUTING.md

```diff
@@ -137,21 +137,29 @@ or
 
 ``pip install -r requirements-dev.txt``
 
-* [OPTIONAL] Set AWS_DEFAULT_REGION to define the region the Data Lake Test envrioment will deploy into. You may want to choose a region which you don't currently use:
+* Go to the ``test_infra`` directory
+
+``cd test_infra``
+
+* Install CDK dependencies:
```

Comment:
Should we just clarify that the CDK needs to be correctly set up and configured? Something along the lines of "we assume CDK is correctly bootstrapped and configured in the AWS account..."